docs/csp.md

```diff
@@ -41,45 +50,58 @@ app.use(
 When loading the initial index page, the `Content-Security-Policy` header should now contain the nonce:
 
 ```
-style-src 'self' 'nonce-N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3'
+default-src 'self'; style-src 'self' 'nonce-N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3';
 ```
 
-2. In webpack config:
+1. Now we want to include this value in our page so that JSS can pick it up at runtime.
+
+You can use any templating engine, or if you have SSR that should work too. For this example we'll use the [Nunjucks](https://github.com/mozilla/nunjucks) template engine.
+
+1. First create the template html file.
+
+```html
+<head>
+  <script data-jss-selector="jss-settings-json" type="application/json">
```

Comment:
type is optional

Reply:
ah, you are using json in here .... I think in that case browser just leaves it as text, its not a type browser will do something with

Reply:
Type is important. If there's no type it will default to `"text/javascript"` which will be blocked by a CSP that prevents inline scripts.

Reply:
actually we can simplify this thing to just use a global variable ...

Reply:
or would this be a leak if we just declare global variable with the nonce value?

Reply:
Which I suppose I should add a comment about.

Reply:
This _is_ possible to do with global value, as dav-is suggested here: https://github.com/cssinjs/jss/issues/559#issuecomment-359685112 
However I was wanting avoid any recommendation of inline javascript, as a matter of preference, as I mentioned here:
https://github.com/cssinjs/jss/issues/559#issuecomment-359800360

Reply:
what about some custom meta tag?

Reply:
We could use an arbitrary tag with a `data-` attribute. 
Example
```html
<span id="jss-config-nonce" data-nonce="NTBkM2U4YmUtYTA5NC00YTBlLTg3NTgtZmZhZDA2ODgxMWVm"></span>
```

Reply:
Could use `<meta ... />` in the head if that's what you meant.

Reply:
sure, I was just thinking what is nicer, meta tag seems nice, also in both cases no JSON.parse, also I think it should be up to user where he graps the nonce text from, and passes it as option, so that we don't have to deal with transport in the core.

Reply:
True... I could go back to using the global variable and document these different ways of setting the nonce. And then as users to pass it to the global variable.

Reply:
for e.g. <meta property="nonce" content="asdfasdfaasdf" />

Reply:
I think meta tag is nicer than global variable

Reply:
also it is more generic, might be used by some other parts of the application as well

Reply:
Ok, I'll refactor to use a `<meta />` tag